e2e/framework/addon/eso.go

```diff
@@ -28,6 +41,39 @@ func NewESO() *ESO {
 	}
 }
 
+func (l *ESO) Install() error {
+	By("Installing eso\n")
+	err := l.Addon.Install()
+	if err != nil {
+		return err
+	}
+
+	By("afterInstall eso\n")
+	err = l.afterInstall()
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (l *ESO) afterInstall() error {
```

Comment:
Shouldn't we allow this one to be more generic to allow for further AWS?

Reply:
If AWS requires any afterinstall steps line annotating something or whatnot, those steps would go there as well. THis is what happens after we install the chart but need to happen before tests start. 

How do you see this being more generic?

Reply:
Ok, pushed something. Is that what you meant?

Reply:
Something like that.. I was afraid Workload identity annotations for GCP could somehow break AWS e2e tests... but I think I tripped hard there.